pages/pil/create/index.js

```diff
@@ -1,11 +1,22 @@
 const { page } = require('@asl/service/ui');
+const differenceInYears = require('date-fns/difference_in_years');
 
 module.exports = settings => {
   const app = page({
     ...settings,
     root: __dirname
   });
 
+  app.use((req, res, next) => {
+    const over18 = req.profile.dob ? differenceInYears(new Date(), new Date(req.profile.dob)) >= 18 : false;
+
+    if (!over18) {
+      throw new Error('All licence applicants must be over 18.');
```

Comment:
This should only happen if they somehow directly hit the route as there should be no "Apply for PIL" button available to them.

Taking that into account, is this error message adequate, or should we reproduce all the variants we have in the profile page (e.g. no dob set, dob set but under 18, own profile, not own profile, etc)?

Reply:
This message seems fine to me, especially as it is highly unlikely to ever be seen